### ISSUE TITLE
Train SSD 600

### ISSUE THREAD

**Issue by paolomanchisi** [GustavZ/realtime_object_detection#14]
HI, I want to train my custom dataset with ssd 600. Is it enough to set 
```
 image_resizer {
      fixed_shape_resizer {
        height: 300
        width: 300
      }
    }
```
to 600 / 600 instead of 300 / 300 ?
Thank you very much.

**Comment by GustavZ** [comment]
yes that is enough, but then you won't be able to use the pre-trained checkpoints.
As the weights only fit a model that takes the same input dimensions!
